intercom_flutter_web/pubspec.yaml

```diff
@@ -24,4 +24,5 @@ dev_dependencies:
   mockito: ^5.4.0
 
 environment:
-  sdk: ">=3.0.0 <4.0.0"
\ No newline at end of file
+  sdk: ">=3.0.0 <4.0.0"
+  flutter: ">=2.0.0"
```

Comment:
Why is this `flutter: ">=2.0.0"` line needed?

Reply:
In this case, since Dart SDK 3 comes bundled with Flutter 3.10, I think this line should be:
```suggestion
  flutter: ">=3.10.0"
```